recipe/meta.yaml

```diff
@@ -14,43 +14,60 @@ source:
     - 0001-Rename-setuptools-scm-to-setuptools_scm.patch
 
 build:
-  number: 0
-  script: python setup.py install --single-version-externally-managed --record record.txt
+  number: 1
+  # This package isn't included in the SOW Packages
+  skip: True  # [s390x or py>=39]
+  script:
+    - export DISABLE_BCOLZ_AVX2=""  # [unix]
+    - export DISABLE_BCOLZ_SSE2=""  # [ppc64le]
+    - set DISABLE_BCOLZ_AVX2=""  # [win]
+    - python setup.py install --single-version-externally-managed --record record.txt
 
 requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - patch  # [not win]
+    - m2-patch  # [win]
   host:
     - python
-    - setuptools >18.0
-    - setuptools_scm >1.5.4
     - cython >=0.22
     - numpy
-  build:
-    - {{ compiler('c') }}
-    - {{ compiler('cxx') }}
+    - pip
+    - setuptools >18.0
+    - setuptools_scm >1.5.4
+    - wheel
   run:
-    - python
+    # It seems like bcolz 1.2.1 isn't compatible with python 3.9+
+    - python <3.9
```

Comment:
The `skip` selector for `py>=39` should be enough to not build this for Python 3.9+. Setting it here might cause odd metadata for the Python 3.7 and 3.8 builds. 🤔

Reply:
I fixed it and moved a comment to the` build/skip`